addons/website_event_booth/views/event_event_views.xml

```diff
@@ -9,10 +9,10 @@
             <field name="address_id" position="after">
                 <field name="exhibition_map"/>
             </field>
-            <field name="website_menu" position="after">
+            <xpath expr="//label[@for='website_menu']" position="after">
```

Comment:
Same if you add any other xpath for label.
```suggestion
            <label for="..." position="after">
```

Reply:
Done :+1: